automation/jenkins_build.sh

```diff
@@ -18,6 +18,46 @@ cleanup() {
 }
 trap 'cleanup fail' SIGINT SIGTERM
 
+deploy_build () {
+	local DEPLOY_DIR="$1"
+	local CHECK_COMPRESSED="$2"
+
+	local DEPLOY_ARTIFACT=$(jq --raw-output '.yocto.deployArtifact' $DEVICE_TYPE_JSON)
+	local COMPRESSED=$(jq --raw-output '.yocto.compressed' $DEVICE_TYPE_JSON)
+	local ARCHIVE=$(jq --raw-output '.yocto.archive' $DEVICE_TYPE_JSON)
+
+	rm -rf "$DEPLOY_DIR"
+	mkdir -p "$DEPLOY_DIR/image"
+
+	cp -v "$YOCTO_BUILD_DEPLOY/VERSION" "$DEPLOY_DIR"
+	cp -v "$YOCTO_BUILD_DEPLOY/VERSION_HOSTOS" "$DEPLOY_DIR"
+	cp -v "$DEVICE_TYPE_JSON" "$DEPLOY_DIR/device-type.json"
+
+	if [ $SLUG != "edge" ]; then
```

Comment:
It would make the code way more readable if this if was something like:
if [ $SLUG = "edge" ]; then
 return
fi